packages/manager/modules/nutanix/src/dashboard/nodes/module.js

```diff
@@ -17,6 +19,7 @@ angular
     'pascalprecht.translate',
     'ui.router',
     node,
+    region,
   ])
   .config(routing)
   .component('nutanixNodes', component)
```

Comment:
```suggestion
  .component('nutanixNodesComponent', component)
```

Reply:
Change not related to this PR. Need to do a refactoring to make this change. It need to be done as part of a new PR.